src/app/organisations/postConfirmEditOrganisation.js

```diff
@@ -0,0 +1,37 @@
+const { editOrganisation } = require("../../infrastructure/organisations");
+const logger = require("../../infrastructure/logger");
+
+const {
+  getOrganisationByIdV2,
+} = require("./../../infrastructure/organisations");
+
+const postConfirmEditOrganisation = async (req, res) => {
+  const correlationId = req.id;
+  const organisation = await getOrganisationByIdV2(req.params.id, req.id);
+  if (!req.session.formData) {
+    return res.redirect(`/organisations/${organisation.id}/users`);
+  }
+  const { name, address } = req.session.formData;
+
+  const body = {
+    name,
+    address,
+  };
+
+  logger.info(`About to update organisation`, {
```

Comment:
This, as is, isn't very useful if you're looking at the logs after the fact.  Something like `About to update organisation with id: [<an org id>]` would be much more helpful.